pygmy/data/qrcode-updater.py

```diff
@@ -0,0 +1,44 @@
+import qrcode
+from base64 import b64encode
+from io import BytesIO
+import sqlite3
+
+def lookForNull():
+    print("Looking for null values")
+    try:
+        conn = sqlite3.connect("pygmy-update.db")
+        cur = conn.cursor()
+        cur.execute('SELECT id, qr_code, short_code FROM link WHERE qr_code IS NULL')

```

Comment:
The query should be updated from `NULL` to `None`.

Right [here](https://github.com/Herrmann-AG/herme-li/pull/6/files#diff-9de1e26b4e1e274fc6370e7a8dd14216b94b01cd4cbae3bb3e337accda7508f9R33) you set the default value for `qr_code` to `None`. So if manually upgrading and sticking to your definition, the upgrade script will fail.

Reply:
The NONE value sets NULL in the database thus i have to filter with "IS NULL". I checked if i can filter with NONE but that didn't worked, it only worked with NULL.

Reply:
Seemed to work for me on my machine, but should not really matter when deploying anyway 🤷🏻‍♂️